demisto_sdk/commands/content_graph/common.py

```diff
@@ -51,7 +51,7 @@ class ContentType(str, enum.Enum):
     INDICATOR_TYPE = 'IndicatorType'
     INTEGRATION = 'Integration'
     JOB = 'Job'
-    LAYOUT = 'Layouts'
+    LAYOUT = 'Layout'
```

Comment:
Is it `Layout` or `Layouts`?

Reply:
layout

Reply:
We use the content type in singular form